app/views/steps/completion/what_next/show.en.html.erb

```diff
@@ -1,81 +1,74 @@
-         and move class `app-list--icon-steps` to the `ol` element -->
-    <section class="app-list--icon-steps">
-      <h2 class="visuallyhidden">Steps</h2>
-      <div class="govuk-govspeak gv-s-prose">
-        <ol class="list list-number">
-          <li class="util_mt-medium">
-            <h3 class="heading-medium">Download your application</h3>
-            <p>
-              This will be a PDF. If you cannot open the PDF after downloading, try using
-              <a href="https://get.adobe.com/uk/reader/" rel="external" target="_blank">Adobe Acrobat Reader</a>.
-            </p>
-
-            <% unless user_signed_in? %>
-              <div class="panel panel-border-narrow">
-                <p>To avoid losing your application, you must download it now. You cannot return to a completed application.</p>
-              </div>
-            <% end %>
+    <h2 class="visuallyhidden">Steps</h2>
+    <div class="govuk-govspeak gv-s-prose">
+      <ol class="list list-number app-list--icon-steps">
+        <li class="util_mt-medium">
+          <h3 class="govuk-heading-m">Download your application</h3>
+          <p class="govuk-body">
+            This will be a PDF. If you cannot open the PDF after downloading, try using
+            <a href="https://get.adobe.com/uk/reader/" rel="external" target="_blank">Adobe Acrobat Reader</a>.
+          </p>
 
-            <p>
-              <%= link_to 'Download your PDF application',
-                          steps_completion_summary_path(format: :pdf),
-                          class: 'button ga-pageLink',
-                          data: {ga_category: 'completion', ga_label: 'download application'} %>
-            </p>
-          </li>
+          <% unless user_signed_in? %>
+            <div class="govuk-inset-text">
+              To avoid losing your application, you must download it now. You cannot return to a completed application.
+            </div>
+          <% end %>
 
-          <li class="util_mt-medium">
-            <h3 class="heading-medium">Print your application</h3>
-            <p>Print 3 copies of your whole application on A4 paper, single sided. The court will send one copy to the other
-              person, keep one copy, and return one copy to you.</p>
-          </li>
+          <p class="govuk-body">
+            <%= link_to 'Download your PDF application',
+                        steps_completion_summary_path(format: :pdf),
+                        class: 'button ga-pageLink',
+                        data: {ga_category: 'completion', ga_label: 'download application'} %>
+          </p>
+        </li>
 
-          <% if current_c100_application.applicants.under_age? %>
-            <li class="util_mt-medium">
-              <h3 class="heading-medium">
-                Send certificate of suitability or court order appointing a litigation friend
-              </h3>
+        <li class="util_mt-medium">
```

Comment:
All the `util_mt-medium` must go, these are old classes to add margins to the top, with the new design system we should not need them.
For an example of numbered (with icons) list, take a look for example to:
`app/views/entrypoint/v1.en.html.erb`